screen_tabs.py

```diff
@@ -339,6 +355,10 @@ def draw(self, context):
 					if stat_value and scene.stat_flags[i]:
 						if key == 'Mem:':
 							stat_value = str(round(float(stat_value[:-1]))) + 'M'
+						if key == 'Verts:' and context.mode == 'EDIT_ARMATURE':
+							val	=	'BONE_DATA'
+						if key == 'Bones:' and context.mode == 'POSE':
```

Comment:
Key **Bones:** already in **stats_icons** dict, no need to check it.

Reply:
Those checks are just for changing the displayed icon.

Reply:
> 
> 
> Those checks are just for changing the displayed icon.

You added **('Bones:', 'ARMATURE_DATA')**, so it is already known that if key == 'Bones:' then use 'ARMATURE_DATA' icon.

Reply:
Object mode:
    objects and meshes/faces/triangles/etc.  Nothing about the armature is displayed.
Edit mode:
    Number of bone points displayed = BONE_DATA
    Number of bones selected and available in the rig  = ARMATURE_DATA
    Rig's name = (the orange armature icon)
Pose mode:
    Number of pose bones selected and available in the rig = BONE_DATA

The reason Pose mode and Edit mode does not both share the same icon, is a design choice. 
Bone points in edit mode are read as vertices, but they're not vertices. 
    By default, they would be displayed using the mesh-vertices icon. 

Solution is to display them using a bone icon. 
The conflict is there isn't another bone icon to be alternatively displayed for pose bones (I was thinking of a blue bone icon).

Reply:
I don't understand the difference - bone is a bone whatever it is pose mode or edit mode.

Reply:
Bones have heads and tails. 
In Edit mode, you can select the _head ***OR*** the tail_.  If you select both, blender will recognize that the entire bone is selected. 
You can ***only*** select the entire bone in Pose mode. 

The default statistics show both bone "points", and "whole" bones in edit mode.

Since you can't select heads/tails in pose mode, the bone "points" are not displayed.

Reply:
I understand what you trying to say. But it is inconsistent to show same values with different icons (bones).